src/open_inwoner/configurations/admin.py

```diff
@@ -29,6 +35,29 @@ class SiteConfigurationPageInline(OrderedTabularInline):
     autocomplete_fields = ("flatpage",)
 
 
+class SiteConfigurarionAdminForm(forms.ModelForm):
+    class Meta:
+        model = SiteConfiguration
+        fields = "__all__"
+
+    def clean(self):
+        cleaned_data = super().clean()
+        redirect_to = cleaned_data.get("redirect_to")
+
+        if redirect_to:
+            if redirect_to.startswith("/"):
+                try:
+                    resolve(redirect_to)
+                except Resolver404:
+                    raise ValidationError(_("The entered path is invalid."))
+            else:
+                validate_url = URLValidator()
+                try:
+                    validate_url(redirect_to)
+                except exceptions.ValidationError:
+                    raise ValidationError(_("The entered url is invalid."))
```

Comment:
It'd be nice to have the error message related to the field: `def clean_redirect_to()`